server/utils/db/db_utils.py

```
@@ -998,6 +1000,21 @@ def initialize_tables():
                 conn.rollback()
 
+            # Add region column to user_connections for multi-account support
+            try:
+                cursor.execute(
+                    "ALTER TABLE user_connections ADD COLUMN IF NOT EXISTS region VARCHAR(50);"
+                )
+                conn.commit()
+                logging.info(
+                    "Ensured region column exists on user_connections table."
+                )
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)